### ISSUE TITLE
Question on RSSI impact on accuracy? 

### ISSUE THREAD

**Issue by IoTThinks** [schollz/find3#199]
Hi all,
Thanks for the awesome project.
I understand that the RSSI will be taken into consideration for machine learning.

In this URL https://www.internalpositioning.com/doc/faq.md#minimum, it is mentioned that the graphs are "similar".

For a layman, these graphs are actually not similar to me.
They are similar with the same detected access points (SSID / BSSID)
However, the RSSI and power are different?

Hence, in this case, the system "should" differentiate the locations of the two graphs, right?
Any advice is appreciated.
Thanks a lot. 

![image](https://user-images.githubusercontent.com/29994971/85390665-516a2a00-b573-11ea-8f5d-538f91068bca.png)

**Comment by schollz** [comment]
Those two graphs are pretty similar. Yello, red and blue are all overlapping the same regions. Only green is very different, but not by much.

**Comment by IoTThinks** [comment]
@schollz Yes, you are right. If we only consider the X-axis then yellow, red and blue are quite the same in two graphs.

BTW what is the unit of the Y-axis? 
The values of Y-axis are different. 
Hence, the "shapes" in two graphs are different. 
Except the yellow graph, the rest have different shape in two graphs.

Thanks a lot for replying me. :)
